fitnesse/resources/bootstrap-plus/templates/skeleton.vm

```diff
@@ -80,10 +80,41 @@
    #parse( $headerTemplate )
   </header>
 
+
   <article>
+   #if(!$bodyClass.equals("edit"))
```

Comment:
de body class van een editor-scherm is editPage. Nu komen de knoppen ook op de edit-pagina